docs/Pitfalls.asciidoc

```diff
@@ -1,16 +1,35 @@
-openQA pitfalls
-===============
-:author: openSUSE Team at SUSE
-:toc:
 
+= openQA pitfalls
+:toc: left
+:toclevels: 6
+:author: OpenQA Team
 
-Needle editing
-==============
 
-- if a new needle is created based on a failed test, the new needle
+== Needle editing
+
+- If a new needle is created based on a failed test, the new needle
   will not be listed in old tests.
-- if an existing needle is updated with a new image or different
+- If an existing needle is updated with a new image or different
   areas, the old test will display the new needle which might be
   confusing
-- if a needle is deleted, old tests may display an error when viewing
+- If a needle is deleted, old tests may display an error when viewing
   them in the web UI.
+
+== Mixed production and Development environment
+
+There are few things to take into account when running a development version and
+a packaged version of openqa:
+
+If the setup for the development scenario involves sharing /var/lib/openqa,
+it would be wise to have a shared group _openqa_, that will have write and execute
+permissions over said directory, so that _geekotest_ user and the normal development
+user can share the environment without problems.
+
+This approach will lead to a problem when the openqa package is updated, since the
+directory permissions will be changed again, nothing a chmod or chgrp can't fix.
+
+[source,bash]
+----
+$ chmod -R g+rwx /var/lib/openqa
+$ chgrp -R openqa /var/lib/openqa
+----
```

Comment:
that's an unrelated change and I suggest to keep it around for later, first let's get the restructuring finished.